packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/templates/generatedAnswer.html

```diff
@@ -22,11 +22,13 @@
               hide-labels
               data-cy="generated-answer__feedback"
             ></c-quantic-feedback>
-            <c-quantic-generated-answer-copy-to-clipboard
-              data-cy="generated-answer__copy-to-clipboard"
-              answer={answer}
-              class="slds-var-m-horizontal_xx-small"
-            ></c-quantic-generated-answer-copy-to-clipboard>
+            <template lwc:if={showCopyToClipboard}>
+              <c-quantic-generated-answer-copy-to-clipboard
```

Comment:
No console error or tooltip text change in quantic?

Reply:
The quantic button fallbacks on the deprecated `document.execCommand('copy')` when the clipboard API fails.  I opted to leave it as is to avoid side effects but, at some point, the fallback will stop working.